scss/bitstyles/tools/_font-size.scss

```diff
@@ -0,0 +1,42 @@
+@use 'sass:map';
+@use '../settings/typography';
+@use 'unit-conversions';
+
+// Palette
```

Comment:
👀 don't really see how this relates to the file name or function 🤔

Reply:
Yup, thanks! Copy/paste error